locales/en/breach-checklist.ftl

```diff
@@ -0,0 +1,57 @@
+# links
+-link-firefox-password-manager = <a href="https://www.mozilla.org/en-US/firefox/features/password-manager/">Firefox Password Manager</a>
+-link-mozilla-vpn = <a href="https://www.mozilla.org/en-US/products/vpn/">Mozilla VPN</a>
+-link-firefox-relay = <a href="https://relay.firefox.com/">Firefox Relay</a>
+-link-equifax = <a href="https://www.equifax.com/personal/credit-report-services/credit-freeze/">Equifax</a>
+-link-experian = <a href="https://www.experian.com/freeze/center.html">Experian</a>
+-link-transunion = <a href="https://www.transunion.com/credit-freeze">TransUnion</a>
+
+# password
+breach-checklist-pw-header = Go to {$companyLink} to change your password and enable two-factor authentication (2FA). 
```

Comment:
When there is a variable, please provide a comment explaining what replaces it at run-time.

Reply:
Done, thanks!